ldap-client.go

```diff
@@ -41,7 +45,10 @@ func (lc *LDAPClient) Connect() error {
 				return err
 			}
 		} else {
-			l, err = ldap.DialTLS("tcp", address, &tls.Config{InsecureSkipVerify: false})
+			l, err = ldap.DialTLS("tcp", address, &tls.Config{
+				InsecureSkipVerify: lc.InsecureSkipVerify,
```

Comment:
When `UseSSL` is true, you **must** either specify `ServerName` or enable `InsecureSkipVerify`.

Reference: https://golang.org/pkg/crypto/tls/#Client